server/schedulers/utils.go

```diff
@@ -333,29 +332,27 @@ func (infl Influence) add(rhs *Influence, w float64) Influence {
 
 // TODO: merge it into OperatorInfluence.
 type pendingInfluence struct {
-	op           *operator.Operator
-	from, to     uint64
-	origin       Influence
-	scheduleTime uint64
-	balanceType  BalanceType
+	op          *operator.Operator
+	from, to    uint64
+	origin      Influence
+	balanceType BalanceType
 }
 
 func (p *pendingInfluence) isDone() bool {
 	return p.op.IsEnd()
 }
 
 func (p *pendingInfluence) isTransferLeader() bool {
-	return strings.Contains(p.op.Desc(), "transfer-leader")
+	return p.op.Kind() == operator.OpLeader|operator.OpAdmin
```

Comment:
Why `OpAdmin`?

Reply:
how about `p.op.Kind()^operator.OpLeader != 0`

Reply:
What does this function want to do? I'm confused.

Reply:
To judge whether `pendingInfluence` is created by `transferLeader`, I guess.

Reply:
The origin PR use `"transfer"` instead of `"transfer-leader"`. It is trying to whether the operator is "transfer-hot-read-leader" or "transfer-hot-write-leader". So, why don't we add a `bool` in `pendingInfluence` to describe this?

Reply:
Well